### PR TITLE
Coreutils suite

### PR DESCRIPTION
**Pull request by kapkic**
This PR adds a bash-coreutils suite, with a very similar structure to our lind suite.
`./coreutils_suite.sh bashtests.txt -v` would execute all the tests in bashtests.txt file. 
If you want to add a new case, make sure you use the full path for the binary, _/bin/mkdir rather than just mkdir_.

### CODE REVIEW

File: tests/test_cases/coreutils_suite/coreutils_suite.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+if [  "$error" == 0 ]; then
+    echo "All tests passed.";
+else
+    echo "Some tests have failed.";
+    echo "$detfails deterministic tests and $nondetfails non-deterministic tests have failed."
+    echo "Failed tests:"
+    echo "${failarray[*]}"
+fi;
+
+exit $error
```
Comment (rennergade) [inline tests/test_cases/coreutils_suite/coreutils_suite.sh:95]: newline

**Review by rennergade — APPROVED** [review]
Overall looks great. Just added a very minor comment.

I would also like to add in the pipescript here, and probably the popen test? We can make that a second PR though.

**Comment by rennergade** [comment]
We should add this to the Makefile to run as well though.

File: tests/test_cases/coreutils_suite/coreutils_suite.sh (hunk 1 above)
Comment (kapkic) [reply tests/test_cases/coreutils_suite/coreutils_suite.sh]: Resolved.

**Review by rennergade — APPROVED** [review]

**Pull request merged**